Podfile

```diff
@@ -12,6 +12,7 @@ pod 'NSURL+IDN', '0.3'
 pod 'DTCoreText',   '1.6.13'
 pod 'UIDeviceIdentifier', '~> 0.1'
 pod 'SVProgressHUD', '~>1.1.3'
+pod 'SDWebImage', '~>3.6'
```

Comment:
Good idea.  I know its a topic some of us have talked about, @sendhil and @oguzkocer in particular. A post we can point back to would be nice.